<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Rohits-MacBook-Pro.local" failures="1" tests="2" name="Test" time="36.718" errors="0" timestamp="2025-02-17T02:11:09 ACDT">
  <testcase classname="tests.AssessmentTest" name="verifyMandatoryField" time="5.225"/>
  <testcase classname="tests.AssessmentTest" name="verifySuccessfullSubmission" time="11.747">
    <failure type="java.lang.AssertionError" message="expected [Thanks Rohit, we appreciate your feedbacks.] but found [Thanks Rohit, we appreciate your feedback.]">
      <![CDATA[java.lang.AssertionError: expected [Thanks Rohit, we appreciate your feedbacks.] but found [Thanks Rohit, we appreciate your feedback.]
at tests.AssessmentTest.verifySuccessfullSubmission(AssessmentTest.java:110)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifySuccessfullSubmission -->
</testsuite> <!-- Test -->
